source/templates.tex

```diff
@@ -3704,7 +3708,8 @@
 
 \pnum
 A name refers to the
-\term{current instantiation}
+\indextext{current instantiation|see{instantiation, current}}
+\defnx{current instantiation}{instantiation!current}
```

Comment:
I don't think grouping this under "instantiation" is appropriate; the "current instantiation" is a notion separate from instantiation itself (in fact, it's most relevant when checking a template definition prior to performing any instantiation).